doc/source/examples/sharp-immersed-boundary-solver/sedimentation-64-particles/sedimentation-64-particles.rst

```diff
@@ -0,0 +1,349 @@
+
+* The ``time step`` is set to  0.0025. This time step is small enough to prevent large error due to the time discretization. 
+
+* The ``time end`` is set to  4.0. This is slightly longer than the time needed for all the particles to reach the bottom of the container
+
+
+
+
+Physical properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 0.1
+        set density             = 0.001
+      end
+    end
+
+
+* The ``kinematic viscosity`` is set to  0.1. This value is derived from the case description by dividing :math:`\mu_f` by :math:`\rho_f`.
+
+* The ``density`` is set to 0.001 according to the description of the problem.
+	
+
+FEM
+~~~
+.. code-block:: text
+
+    subsection FEM
+      set velocity order = 1
+      set pressure order = 1
+    end
+	
+Here we use Q1Q1 elements. This case is only for demonstration purposes; as such, we suggest a simulation that is not too costly to run. 
+
+Mesh
+~~~~~~
+.. code-block:: text
+
+    subsection mesh
+        set type                 = dealii
+    	set grid type            = subdivided_hyper_rectangle
+    	set grid arguments       = 1,1,12: 0,0,0 : 2 , 2 , 24 : true
+    	set initial refinement   = 4
+    end
+
+The domain is a rectangular box: we can directly use a subdivided hyper rectangle mesh from the deal.II library. In this case, we have oriented the z-direction with gravity. As such, we have the long side of the box along this axis.
+
+* The ``grid arguments`` is set to  ``1,1,12: 0,0,0 : 2,2,24 : true``. This section has 3 subsections. First ``1,1,12`` describes the initial subdivision of the box. This subdivision has been chosen as it is the smallest mesh we can do of the box in order to have cubic elements. Secondly ``0,0,0 : 2,2,24`` describes the 2 points from which we have derived the rectangular box (0,0,0) and  (2,2,24). Finally, we have ``true``, which is a boolean to activate the coloration of the boundary. This allows us to define separate boundary conditions at each side of the box.
+
+* The ``initial refinement`` is set to 4. This will ensure to have a base mesh that is a bit finer than the particle size.
+
+
+Mesh adaptation control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code-block:: text
+
+    subsection mesh adaptation
+      # Fraction of coarsened elements
+      set fraction coarsening = 0.2
+    
+      # Fraction of refined elements
+      set fraction refinement = 0.025
+    
+      # How the fraction of refinement/coarsening are interepretedChoices are
+      # <number|fraction>.
+      set fraction type = number
+    
+      # Frequency of the mesh refinement
+      set frequency = 1
+    
+      # Maximum number of elements
+      set max number elements = 750000
+    
+      # Maximum refinement level
```

Comment:
This section was taken from the default prm file generated by the template and lightly modified. I could remove them since the parameter are also explained in the rst. But I don't feel like they clutter this section.

Reply:
It would be better if you could remove them. I have not seen any of our other examples which kept comments in the subsection of the rst file. Once that's removed, I'll merge the PR.